sq/src/sq_cli.rs

```diff
@@ -637,6 +637,18 @@ $ sq key generate --userid \"<juliet@example.org>\" --userid \"Juliet Capulet\"
                                  "rev-cert",
                              ])
                              .requires("userid"))
+                        .arg(Arg::with_name("primary-flags")
+                             .long("primary-flags").value_name("[C | CS]")
+                             .help("Generate keys using 2-key or 3-key structure")
+                             .conflicts_with_all(&[
```

Comment:
@zugzwang -- could you double-check this?

Reply:
LGTM

Reply:
resolving to reduce clutter